transports/quic/src/transport.rs

```diff
@@ -49,12 +49,24 @@ use std::{
 };
 
 /// Implementation of the [`Transport`] trait for QUIC.
+///
+/// Per default only QUIC Version 1 (RFC 9000) is supported. In the [`Multiaddr`] this maps to
```

Comment:
```suggestion
/// By default only QUIC Version 1 (RFC 9000) is supported. In the [`Multiaddr`] this maps to
```

Not a native speaker but "per" sounds odd to me.